File: CS/10_Conversion/LoadSaveDPSAndDPT.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;
using Spire.Presentation.Drawing;
using System.IO;

namespace LoadSaveDPSAndDPT
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create Presentation
            Presentation presentation = new Presentation();

            //Load .dps or .dpt file
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\sample.dps",Spire.Presentation.FileFormat.Dps);
             //presentation.LoadFromFile(@"..\..\..\..\..\..\Data\sample.dpt",Spire.Presentation.FileFormat.Dpt);

            //Save the .dps or .dpt file
            String result = "LoadSaveDPSAndDPT_result.dps";
            presentation.SaveToFile(result, Spire.Presentation.FileFormat.Dps);
            //presentation.SaveToFile("LoadSaveDPSAndDPT_result.dpt", Spire.Presentation.FileFormat.Dpt);
            //Launching the result file.
            Viewer(result);
        }
        private void Viewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/10_Conversion/LoadSaveDPSAndDPT.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation
Imports Spire.Presentation.Drawing
Imports System.IO

Namespace LoadSaveDPSAndDPT
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create Presentation
			Dim presentation As New Presentation()

			'Load .dps or .dpt file
			presentation.LoadFromFile("..\..\..\..\..\..\Data\sample.dps",Spire.Presentation.FileFormat.Dps)
			 'presentation.LoadFromFile(@"..\..\..\..\..\..\Data\sample.dpt",Spire.Presentation.FileFormat.Dpt);

			'Save the .dps or .dpt file
			Dim result As String = "LoadSaveDPSAndDPT_result.dps"
			presentation.SaveToFile(result, Spire.Presentation.FileFormat.Dps)
			'presentation.SaveToFile("LoadSaveDPSAndDPT_result.dpt", Spire.Presentation.FileFormat.Dpt);
			'Launching the result file.
			Viewer(result)
		End Sub
		Private Sub Viewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace